pkg/selectors/kernel.go

```diff
@@ -576,45 +578,55 @@ func getBase(v string) int {
 }
 
 func parseAddr(v string) ([]byte, uint32, error) {
-	ipaddr := net.ParseIP(v)
-	if ipaddr != nil {
-		ipaddr4 := ipaddr.To4()
-		if ipaddr4 != nil {
-			return ipaddr4, 32, nil
-		}
-		ipaddr6 := ipaddr.To16()
-		if ipaddr6 != nil {
-			return ipaddr6, 128, nil
-		}
-		return nil, 0, errors.New("IP address is not valid: does not parse as IPv4 or IPv6")
-	}
+	var maskLen uint32
+
 	vParts := strings.Split(v, "/")
```

Comment:
Can we rip most of this out and use net.ParseCIDR() instead? https://pkg.go.dev/net#ParseCIDR

Reply:
Hm, I thought that `net.ParseCIDR()` will not work correctly with addresses like `::ffff:127.0.0.1/16`, but I was wrong, it returns correct mask of total length of 128 bits.
I'll try to apply this.